open-telemetry-addon/hack/certs/san.cnf

```diff
@@ -0,0 +1,13 @@
+[ req ]
+distinguished_name = req_distinguished_name
+req_extensions = v3_req
+prompt = no
+
+[ req_distinguished_name ]
+CN = hub-control-plane
+
+[ v3_req ]
+subjectAltName = @alt_names
+
+[ alt_names ]
+DNS.1 = hub-control-plane
```

Comment:
should this be changed for different hub cluster?

Reply:
It is a KinD cluster name. We do not need to make change if use quick start guide to setup a test environment. We can expose it when we have a helm chart for this addon.